Kuroba/app/src/main/res/values/strings.xml

```diff
@@ -646,139 +646,139 @@ Don't have a 4chan Pass?<br>
     <string name="import_or_export_dialog_positive_button_text">Overwrite</string>
     <string name="import_or_export_dialog_negative_button_text">Create new</string>
     <string name="import_or_export_warning">Warning!</string>
-    <string name="import_or_export_warning_super_long_message_part_one">You have %1$s set to be located in a directory that uses the Storage Access Framework. This directory location cannot be exported; IT WILL BE RESET TO DEFAULT in the exported settings file! Once you import them you will have to manually set those locations in Media Settings again.</string>
+    <string name="import_or_export_warning_super_long_message_part_one">You have %1$s set to be located in a directory that uses the \"Storage Access Framework\". This directory location cannot be exported; IT WILL BE RESET TO DEFAULT in the exported settings file! Once you import them you will have to manually set those locations in \"Media Settings\" again.</string>
     <string name="import_or_export_warning_saved_files_base_dir">\"saved files base directory\"</string>
 
-    <string name="media_settings_use_saf_for_save_location_dialog_title">Use the new Storage Access Framework API to choose images download directory?</string>
-    <string name="media_settings_use_saf_for_save_location_dialog_message">If you choose to use the SAF you will be able to choose an SD-card as a place to store downloaded images</string>
+    <string name="media_settings_use_saf_for_save_location_dialog_title">Use the new \"Storage Access Framework API\" to choose where to download images?</string>
+    <string name="media_settings_use_saf_for_save_location_dialog_message">Allows downloading images to SD card.</string>
     <string name="media_settings_use_saf_dialog_positive_button_text">Use SAF API</string>
     <string name="media_settings_use_saf_dialog_negative_button_text">Use old API</string>
     <string name="media_settings_could_not_copy_files">Could not copy one directory\'s file into another one</string>
-    <string name="media_settings_files_copied">Successfully copied files</string>
-    <string name="media_settings_would_you_like_to_delete_file_in_old_dir">Would you like to delete files in the old directory?</string>
-    <string name="media_settings_file_have_been_copied">Files have been copied and now exist in two directories. You may want to remove files in the old directory \n\n\"%1$s\"\n\nsince you won\'t need them anymore. \nThis operation will DELETE ALL the files in that directory, so if you had some files not related to this app you probably should do that manually.</string>
-    <string name="media_settings_could_not_delete_files_in_old_dir">Couldn\'t delete files in the old directory</string>
-    <string name="media_settings_old_files_deleted">Successfully deleted files in the old directory</string>
+    <string name="media_settings_files_copied">All files were copied</string>
```

Comment:
``Files copied successfully``?

Reply:
"Successfully" implies it could be done in non-successful fashion, or that it usually (or likely) isn't successful, neither of which is the case. It is fully redundant, and will make its way into translations to dubious effect.